packages/babel-plugin-proposal-logical-assignment-operators/src/index.js

```diff
@@ -39,11 +39,19 @@ export default declare(api => {
           }
         }
 
+        const isRHSAnonymousorArrowFunction =
+          t.isFunctionExpression(right, {
```

Comment:
Nit: Can't this just be `t.isFunction`? That should capture both arrow and regular function expressions.

Reply:
@jridgewell My bad. Missed this comment.